include/convex_bodies/hpolytope.h

```diff
@@ -99,20 +99,20 @@ class HPolytope {
 
             NT const tol = 0.00000001;
             std::tuple<VT, NT, bool> inner_ball = max_inscribed_ball(A, b, 150, tol);
-        
```

Comment:
are those changes by mistake?

Reply:
I think this is due to another branch that has merged these changes into develop.